test/concerns/tree_predicate_test.rb

```diff
@@ -1,8 +1,8 @@
-require_relative '../environment'
+require_relative "../environment"
 
 class TreePredicateTest < ActiveSupport::TestCase
   def test_tree_predicates
-    AncestryTestDatabase.with_model :depth => 2, :width => 3 do |_model, roots|
+    AncestryTestDatabase.with_model depth: 2, width: 3 do |_model, roots|
```

Comment:
all the other code uses hash rocket syntax.
Could you remove this change from this PR?

Reply:
sorry, my editor auto-applies that transform; will undo.

Reply:
@Roguelazer could you share the editor / plugin to do that. sounds useful

Thanks for the PR

Reply:
I use [conform.nvim](https://github.com/stevearc/conform.nvim) to run a subset of rubocop auto-fixes